lib/absinthe/execution/resolution/field.ex

```diff
@@ -9,22 +9,19 @@ defimpl Absinthe.Execution.Resolution, for: Absinthe.Language.Field do
 
   @spec resolve(Absinthe.Language.Field.t,
                 Absinthe.Execution.t) :: {:ok, map} | {:error, any}
-  def resolve(%{name: name} = ast_node, %{strategy: :serial, resolution: %{parent_type: parent_type, target: target}} = execution) do
+  def resolve(%{name: name} = ast_node, %{strategy: :serial, resolution: %{parent_type: parent_type, target: target}, schema: schema} = execution) do
     field = find_field(ast_node, execution)
-    case field do
-      %{resolve: nil} ->
+    case {field, schema.__absinthe_custom_default_resolve__} do
```

Comment:
Yes, because if no _custom_ default resolver is defined, we can skip the overhead associated with argument and field info building and just resolve immediately using the built-in logic below.